.changeset/silent-cobras-run.md

```diff
@@ -0,0 +1,16 @@
+---
+"@aws-amplify/ui-react-liveness": patch
```

Comment:
should this be a minor? since we're releasing a new feature

Reply:
+1